BUILD.bazel

```diff
@@ -534,17 +534,24 @@ cc_library(
             "src/ray/gcs/gcs_server/gcs_server_main.cc",
             "src/ray/gcs/gcs_server/test/*.cc",
         ],
-    ),
+    ) + [
+        "src/ray/raylet/scheduling/local_resource_manager.cc",
```

Comment:
should we use a separate target for this and use it in both raylet and gcs ?

Reply:
> should we use a separate target for this and use it in both raylet and gcs ?

Yes, @scv119 is doing this. see #23124